scripts/make/install-drupal.sh

```diff
@@ -12,12 +12,12 @@ if [ "$drupal_build_composer_install" == "Y" ]; then
   composer self-update && composer install
 fi
 
+if [ "$drupal_fix_settings" == "Y" ]; then
+  chmod u+w docroot/sites/* docroot/sites/*/settings.php
+fi
+
 if [ "$drupal_build_drush_make" == "Y" ]; then
   cd docroot \
     && ../vendor/bin/drush @none make -y --nocolor --no-recursion ../drush-make.yml
   cd ..
 fi
-
```

Comment:
I see no problem with checking and fixing permissions prior to running Drush Make.
We still need to run the check and fix permissions after Drush Make has ran though because Drupal insists of changing directory and file permissions of settings as part of Drupal installation.

Reply:
ok, updated the pull request to run it twice.